### ISSUE TITLE
High resolution image not converge

### ISSUE THREAD

**Issue by LaFeuilleMorte** [nyu-systems/Grendel-GS#31]
Hi, I'm using this model to train on 8K images dataset. But the loss won't converge. And the PSNR, SSIM metrics are much worse than in 2K experiments. How can I adjust the hyperparameters to get better results in much higher resolution.

**Comment by TarzanZhao** [comment]
Hi @LaFeuilleMorte , I do not know the reason only from these description. There can be many potential reasons. We can schedule a meeting if you want to diagnose this issue together: hz3496@nyu.edu

**Comment by LaFeuilleMorte** [comment]
Sure， which time zone are you thereSent from my iPhoneOn Oct 17, 2024, at 02:25, Hexu Zhao ***@***.***> wrote:﻿
Hi @LaFeuilleMorte , I do not know the reason only from these description. There can be many potential reasons. We can schedule a meeting if you want to diagnose this issue together: ***@***.***

**Comment by LaFeuilleMorte** [comment]
> Sure， which time zone are you thereSent from my iPhoneOn Oct 17, 2024, at 02:25, Hexu Zhao ***@***.***> wrote:﻿ Hi @LaFeuilleMorte , I do not know the reason only from these description. There can be many potential reasons. We can schedule a meeting if you want to diagnose this issue together: ***@***.*** —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

Hi, I think I've known what't going wrong with my experiments. The reason why the reconstruction in 8K is much worse than in 2K was mainly because I forgot to use the undistorted images in 8K resolution. The distortion may introduce huge errors in such a high resolution.

**Comment by TarzanZhao** [comment]
Sorry, I did not receive your email.

**Comment by TarzanZhao** [comment]
Can you send again? We can connect and know each other.

**Comment by glorioushonor** [comment]
> Hi, I'm using this model to train on 8K images dataset. But the loss won't converge. And the PSNR, SSIM metrics are much worse than in 2K experiments. How can I adjust the hyperparameters to get better results in much higher resolution.

Hello, I apologize for the interruption. I am currently training on the Campus dataset and seem to be encountering a very similar issue—my training process is failing to converge, and the final results contain a large number of floating Gaussian spheres. Could you kindly provide me with some advice or guidance on this matter?
![image](https://github.com/user-attachments/assets/a240addc-c3d3-4bbb-b7d0-7b1e7ae0be05)
![image](https://github.com/user-attachments/assets/b394789d-adee-4f06-8bbe-b26cd9babf27)
